Formula/clipboard.rb

```diff
@@ -0,0 +1,40 @@
+class Clipboard < Formula
+  desc "Cut, copy, and paste anything, anywhere, all from the terminal"
+  homepage "https://getclipboard.app"
+  url "https://github.com/Slackadays/Clipboard/archive/refs/tags/0.3.1.tar.gz"
+  sha256 "c00580e60c7c402bb0b8e87133fdc511b25205200adb2c17e96c5811cc79c37d"
+  license "GPL-3.0-or-later"
+  head "https://github.com/Slackadays/Clipboard.git", branch: "main"
+
+  depends_on "cmake" => :build
+  depends_on macos: :monterey
+
+  on_linux do
+    depends_on "pkg-config" => :build
+    depends_on "wayland-protocols" => :build
+    depends_on "libx11"
+    depends_on "wayland"
+  end
+
+  fails_with :clang do
+    build 1300
+    cause "Requires C++20 support"
+  end
+
+  def install
+    args = std_cmake_args
+    args << "-DINSTALL_PREFIX=#{prefix}"
+    system "cmake", "-S", ".", "-B", "build", *args
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
+
+    man1.install "documentation/manpages/man.1" => "clipboard.1"
```

Comment:
Doesn't cmake install this?

Reply:
No, manpage needs to be installed manually

Reply:
Could you make an issue for that upstream because that's a weird thing to leave out of your build system. Makes me think they just forgot about the manpages

Reply:
Was addressed in upstream recently, pending release

Reply:
Done